theme/screens.scss

```diff
@@ -1 +1,2 @@
-$small-desktop: 800px;
+$small-desktop: 600px;
```

Comment:
I find it hard to believe that `600px` is desktop at all. Most of the tablet screens are bigger than this.

Reply:
You can name it `$tablet` after all. 🙂